colab_repoclone/__init__.py

```diff
@@ -0,0 +1,36 @@
+# __init__.py for colab_repoclone
+
+name = "colab_repoclone"
+__version__ = "0.0.1"
+
+import sys
+
+if "google.colab" in sys.modules:
+    import colab_env
+from colab_repoclone.git_access import RepoClone
+
+
+def load_repository(repo, branch="master", method="env"):
```

Comment:
Recommend renaming this to `clone_repository`, since that's what it does.
```suggestion
def clone_repository(repo, branch="master", method="env"):
```